packages/longhorn/package.yaml

```
@@ -0,0 +1,5 @@
+url: https://github.com/longhorn/charts/releases/download/longhorn-1.0.1/longhorn-1.0.1.tgz
```

We should use the git format for the `package.yaml` for this:

```suggestion
url: https://github.com/longhorn/charts.git
subdirectory: charts/longhorn
type: git
commit: longhorn-1.0.1
```